integration-tests/emulated/src/mock/network.rs

```diff
@@ -14,28 +14,32 @@
 // You should have received a copy of the GNU General Public License
 // along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
+use asset_hub_rococo_emulated_chain::AssetHubRococo as AssetHubParachain;
+use rococo_emulated_chain::Rococo as RococoChain;
 use xcm_emulator::decl_test_networks;
 
-use crate::mock::{
-	para_chains::{AssetHubPolkadot, AssetHubRococo, Peregrine, Spiritnet},
-	relay_chains::{Polkadot, Rococo},
-};
+use crate::mock::para_chains::{PeregrineParachain, SpiritnetParachain};
+
+#[cfg(not(feature = "runtime-benchmarks"))]
+#[cfg(test)]
+pub mod chains {
+
+	pub use super::*;
+
+	pub type Rococo = RococoChain<MockNetwork>;
+	pub type Spiritnet = SpiritnetParachain<MockNetwork>;
+	pub type Peregrine = PeregrineParachain<MockNetwork>;
+	pub type AssetHub = AssetHubParachain<MockNetwork>;
+}
 
 decl_test_networks! {
-	pub struct MockNetworkPolkadot {
-		relay_chain = Polkadot,
+	pub struct MockNetwork {
```

Comment:
Do you think it would be better to have two different networks, one with AssetHub + Spiritnet and one with AssetHub + Peregrine? The relaychain would be the same, but we can change it later on, and we can use AssetHub on Paseo for Peregrine.

Reply:
I have the feeling, we are using the emulator kinda wrong. Maybe we should do some research and define what should be the scope of the emulator and chopsticks. I think using two networks is giving no benefits. It would be kinda cleaner, but I am unsure, if the maintaining effort would be worth it.

Reply:
Yeah we'll scrape these tests soon, so let's keep it like this for now.